examples/openshift-compatible/README.md

```diff
@@ -1,13 +1,15 @@
 # OpenShift Compatible
 
+TODO THESE PARAGRAPHS NEED UPDATING
```

Comment:
Still needed?

Reply:
Ahh, geez.. The ironic thing is that I put that there to remind myself to deal with that before comitting.
I need to make a pre-commit hook just for myself...